.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,14 @@
+[Project Reno Jira Ticket](http://jira.nypl.org/browse/RENO-XXX)
+
+[ResearchNow Jira Ticket](http://jira.nypl.org/browse/SFR-XXX)
```

Comment:
Is this file a WIP?  Lots of reno/drupal specific stuff here

Reply:
I tried to include everything for both projects in this PR template with the idea that users can delete what's not relevant. Do you think there's a better way to do this?

Reply:
I think actually for now I'm going to recommend we edit this PR template when we come to it. It's not perfect but feel free to open up an issue to discuss improving it if you would like to discuss this more async, Crystal.